src/libertem/executor/integration.py

```diff
@@ -0,0 +1,38 @@
+import types
+
+import dask
+import dask.delayed
```

Comment:
Good point, good to check! It doesn't start anything since it will just use whatever Dask scheduler is configured when `compute()` is called. The overhead of importing it is zero:

![image](https://user-images.githubusercontent.com/25689052/143866044-d2847247-5b09-4905-a840-acd1312c0476.png)

It appears that Dask loads it somewhen, somehow anyway.